web/src/components/Welcome.tsx

```diff
@@ -74,6 +75,9 @@ const Welcome: React.FC = () => {
         <ListItem>
           <FaLinux /> {t('providers.linux')}
         </ListItem>
+        <ListItem>
+          <FaInstagram /> {t('providers.instagram')}
+        </ListItem>
         <ListItem>
           <FaTwitter /> {t('providers.twitter')}
         </ListItem>
```

Comment:
I prefer putting Instagram card next to Twitter card.

```suggestion
        <ListItem>
          <FaTwitter /> {t('providers.twitter')}
        </ListItem>
        <ListItem>
          <FaInstagram /> {t('providers.instagram')}
        </ListItem>
```